installation/docker.md

```diff
@@ -6,5 +6,68 @@ layout: documentation
 
 # Installation through Docker
 
-to be done...
+## Usage
+
+**Important** To be able to use UPnP for discovery the container needs to be started with ``--net=host``.
+
+The following will run openHAB in demo mode on the host machine:
+```
+docker run -it --name openhab --net=host openhab/openhab:amd64-online server
```

Comment:
Question: "amd64-online" might not always be the right choice, is it?
Would it make sense to mention the other options?